lib/checkpw.c

```diff
@@ -95,6 +95,7 @@
 # endif
 #endif
 
+# include <winldap.h>
```

Comment:
This include clearly breaks Unix platforms. So you need to wrap it in some form of #ifdef

Reply:
should  _WIN32 macro test enough ?  if so I'll update this